app/src/pages/create.tsx

```diff
@@ -116,6 +117,8 @@ function Create() {
   const { status } = useSession();
   const { mutateAsync } = trpc.schedule.createSchedule.useMutation();
   const [isDatePickerOpen, setIsDatePickerOpen] = useAtom(datePickerOpen);
+  const [, setTitle] = useAtom(updateTitle);
```

Comment:
so here you can just do:
```suggestion
  const [, setTitle] = useAtom(pageTitle);
```
and you can replace the rest of the files to use this way